staging/src/k8s.io/kms/internal/plugins/_mock/pkcs11/pkcs11.go

```diff
@@ -111,7 +111,7 @@ func (s *pkcs11RemoteService) Decrypt(ctx context.Context, uid string, req *serv
 
 func (s *pkcs11RemoteService) Status(ctx context.Context) (*service.StatusResponse, error) {
 	return &service.StatusResponse{
-		Version: "v2beta1",
+		Version: "v2",
```

Comment:
This is okay to change since this is our "example" plugin we want people to copy